openapi3/swagger_loader.go

```diff
@@ -342,14 +443,16 @@ func (swaggerLoader *SwaggerLoader) resolveComponent(swagger *Swagger, ref strin
 			return nil, "", nil, fmt.Errorf("Error while resolving path: %v", err)
 		}
 
-		if swg2, ok := swaggerLoader.loadedRemoteSchemas[parsedURL.String()]; !ok {
+		key := parsedURL.String()
+		if swg2, ok := swaggerLoader.loadedRemoteSchemas[key]; !ok {
 			if swg2, err = swaggerLoader.LoadSwaggerFromURI(resolvedPath); err != nil {
 				return nil, "", nil, fmt.Errorf("Error while resolving reference '%s': %v", ref, err)
 			}
-			swaggerLoader.loadedRemoteSchemas[parsedURL.String()] = swg2
+			swaggerLoader.loadedRemoteSchemas[key] = swg2
 		}
-		swagger.Components = mergeComponents(swagger.Components, swaggerLoader.loadedRemoteSchemas[parsedURL.String()].Components)
+		//swagger.Components = mergeComponents(swagger.Components, swaggerLoader.loadedRemoteSchemas[parsedURL.String()].Components)
```

Comment:
Go ahead and remove this comment and the function it refers to.  We can get the data from history if we want it back.